bus/bus.go

```diff
@@ -1532,13 +1551,18 @@ func (b *bus) Handler() http.Handler {
 		"GET    /slab/:key":           b.slabHandlerGET,
 		"PUT    /slab":                b.slabHandlerPUT,
 
+		"POST /search/hosts":   b.searchHostsHandlerPOST,
+		"GET  /search/objects": b.searchObjectsHandlerGET,
+
+		"GET    /stats/objects": b.objectsStatshandlerGET,
+
 		"GET    /settings":     b.settingsHandlerGET,
 		"GET    /setting/:key": b.settingKeyHandlerGET,
 		"PUT    /setting/:key": b.settingKeyHandlerPUT,
 		"DELETE /setting/:key": b.settingKeyHandlerDELETE,
 
-		"GET    /params/upload":  b.paramsHandlerUploadGET,
-		"GET    /params/gouging": b.paramsHandlerGougingGET,
+		"POST /upload/:id":          b.uploadHandlerPOST,
+		"POST /upload/:id/finished": b.uploadFinishedHandlerDELETE,
```

Comment:
I wasn't convinced at first but I guess those routes make sense.

It's actually kinda nice. We could extend it later with the Size and other metadata and have the UI poll it to display ongoing uploads.
Including some info whether it's a regular upload or a migration and the name of the object being uploaded

Reply:
ha! this is actually exactly what I had in mind for these routes!